src/App.jsx

```diff
@@ -1,107 +1,33 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+const posts = postsFromServer.map((post) => {
+  function getUserById(userId) {
+    return usersFromServer.find(user => user.id === userId);
+  }
+
+  function getCommentsByPostId(postId) {
+    return commentsFromServer.filter(comment => comment.postId === postId);
+  }
+
+  return {
+    ...post,
+    user: getUserById(post.userId),
+    comments: getCommentsByPostId(post.id) || [],
```

Comment:
```suggestion
    comments: getCommentsByPostId(post.id),
```
filter always returns array